pronto_quadruped_commons/include/pronto_quadruped_commons/feet_jacobians.h

```diff
@@ -53,18 +54,16 @@ class FeetJacobians
      * takes the identifier of the leg of interest.
      */
     ///@{
-    virtual FootJac getFootJacobianLF(const JointState& q) = 0;
-    virtual FootJac getFootJacobianRF(const JointState& q) = 0;
-    virtual FootJac getFootJacobianLH(const JointState& q) = 0;
-    virtual FootJac getFootJacobianRH(const JointState& q) = 0;
+    virtual FootJac getFootJacobianLF(const JointState& q) { return getFootJacobian(q, LegID::LF); }
```

Comment:
I'm not even sure these functions get called explicitly by the user, I might just remove them from the interface entirely. They were there just because robcogen had a function for each individual leg

Reply:
More than happy to remove them, it simplifies the interface (since `getFootJacobian(q, leg)` does the same thing) - will do!

Reply:
Actually, I won't remove them right now and add a comment for clean-up since it would force changes on all robot implementations.